### ISSUE TITLE
cache implementation

### ISSUE THREAD

**Issue by maggu2810** [openhab/openhab-core#814]
openHAB Core is using its own cache implementation (migrated from ESH).

See e.g.:
* https://github.com/eclipse/smarthome/pull/3572
* https://github.com/eclipse/smarthome/pull/3932

Wouldn't it make more sense to rely on the standard [JSR 107: JCACHE - Java Temporary Caching API](https://www.jcp.org/en/jsr/detail?id=107)?

There is https://github.com/jsr107/jsr107spec

**Comment by kaikreuzer** [comment]
If JCache were part of the JDK, I would clearly say, yes, we should use it.
But as it is just a JSR, originally targeting J2EE, if I see it correctly, I think it does not bring us much benefit. What we have is working and it is under our control. Why put in any efforts to change it?

**Comment by maggu2810** [comment]
As you have written yourself [here](https://github.com/openhab/openhab-core/pull/611#issuecomment-482280535)

> making solutions more future proof by using official standards instead of specific implementations.

There is no need to change it.
We can use what we have and keep it under our control.

IMHO if there are standards, we should use them without reinvent the wheel.

But in this case it is a very little wheel and it has been already reinvented.
So if your point is to stay at our wheel as long as it is working, I can life with it for this little subsystem.

I just started to use `JCache` in another project and wonder what we did not use it...
